pyproject.toml

```diff
@@ -147,13 +147,13 @@ crawler = [
   "selenium>=4.11.0"
 ]
 preprocessing = [
-  "nltk",
+  "nltk>=3.9",
   "langdetect",  # for language classification
 ]
 file-conversion = [
   "azure-ai-formrecognizer>=3.2.0b2",  # Microsoft Azure's Form Recognizer service (text and table exctrator)
   "python-docx",
-  "python-pptx",
+  "python-pptx<=1.0",
```

Comment:
Could you explain this pin?

Reply:
Yes, see [this](https://github.com/deepset-ai/haystack/actions/runs/10469920058/job/28993957628) After pin, lint passed.